JEKScrollableSectionCollectionViewLayout.m

```diff
@@ -349,6 +351,11 @@ - (CGSize)itemSizeForIndexPath:(NSIndexPath *)indexPath
     return [self.delegate respondsToSelector:@selector(collectionView:layout:sizeForItemAtIndexPath:)] ? [self.delegate collectionView:self.collectionView layout:self sizeForItemAtIndexPath:indexPath] : self.itemSize;
 }
 
+- (BOOL)flowLayoutUsedInSection:(NSInteger)section
+{
+    return [self.sectionStyleDelegate respondsToSelector:@selector(collectionView:layout:flowLayoutUsedInSection:)] ? [self.sectionStyleDelegate collectionView:self.collectionView layout:self flowLayoutUsedInSection:section] : NO;
+}
```

Comment:
To be more "objective-c"-ish I suggest renaming this function to `shouldUseFlowLayoutInSection:`!

Also, if you make it part of the already existing protocol, you can change the implementation to:
```objc
if (DELEGATE_RESPONDS_TO_SELECTOR(@selector(collectionView:layout:shouldUseFlowLayoutInSection:)) {
    return [DELEGATE collectionView:self.collectionView layout:self shouldUseFlowLayoutInSection:section];
}
return NO;
```